surveys/forms.py

```diff
@@ -21,17 +24,97 @@ def create_default_helper(self):
         self.helper.add_input(Submit('submit', 'Submit'))
 
 
-class CreateAgencyForm(JustSpacesForm):
+class AgencyCreateForm(JustSpacesForm):
     class Meta:
         model = Agency
         fields = '__all__'
 
 
-class CreateLocationForm(JustSpacesForm):
+class LocationCreateForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(LocationCreateForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper()
+        self.helper.form_class = 'form-horizontal'
+        self.helper.label_class = 'col-lg-2'
+        self.helper.field_class = 'col-lg-8'
+        self.helper.form_method = 'post'
+        self.helper.form_tag = False
```

Comment:
There are two differences here from `JustSpacesForm` that mean, as far as I can see, we can't inherit. Both of them are because the `location_create.html` template actually has three forms (`LocationForm`, `LocationLineForm`, and `LocationAreaForm`) that we want to look like a single form. See [here](https://django-crispy-forms.readthedocs.io/en/latest/crispy_tag_forms.html?highlight=several#rendering-several-forms-with-helpers).

- `self.helper.form_tag = False`: This prevents the helper from adding \<form\> tags to the template
- No `self.helper.add_input(Submit('submit', 'Submit'))`: This means django-crispy-forms won't add a submit button at the bottom of this form

Reply:
Gotcha! I think the duplication is fine here, then.